tests/Neo.Plugins.RpcServer.Tests/UT_RpcServer.SmartContract.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// UT_RpcServer.Wallet.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Neo.IO;
+using Neo.Json;
+using Neo.Network.P2P.Payloads;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+using Neo.UnitTests;
+using Neo.UnitTests.Extensions;
+using Neo.Wallets;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace Neo.Plugins.RpcServer.Tests;
+
+public partial class UT_RpcServer
+{
+    static readonly string neoScriptHash = "0xef4073a0f2b305a38ec4050e4d3d28bc40ea63f5";
+    static readonly string NeoTotalSupplyScript = "wh8MC3RvdGFsU3VwcGx5DBT1Y\u002BpAvCg9TQ4FxI6jBbPyoHNA70FifVtS";
+    static readonly JArray signers = [new JObject()
```

Comment:
can we do abnormal test with tons of signers, tons of opened sessions, invalid address